pkg/test/nodeclaim.go

```diff
@@ -48,6 +48,7 @@ func NodeClaim(overrides ...v1beta1.NodeClaim) *v1beta1.NodeClaim {
 	if override.Spec.Requirements == nil {
 		override.Spec.Requirements = []v1beta1.NodeSelectorRequirementWithMinValues{}
 	}
+	override.Spec.StartupTaints = append(override.Spec.StartupTaints, v1beta1.UnregisteredNoExecuteTaint)
```

Comment:
This seems unexpected to me. Since we are artificially adding this taint from the CloudProvider side, is it possible that the node just registers with this taint, even if the NodeClaim didn't store it?

Reply:
No longer going this route.